src/Standards/Generic/Tests/PHP/DisallowShortOpenTagUnitTest.php

```diff
@@ -52,13 +52,6 @@ public function getErrorList($testFile='')
     {
         switch ($testFile) {
         case 'DisallowShortOpenTagUnitTest.1.inc':
-            if (PHP_VERSION_ID < 50400) {
```

Comment:
I think more changes are needed for the DisallowShortOpenTag sniff. I'll have a look.

Reply:
@jrfnl Did you have chance to check it? I don't know what do you mean...

Reply:
@webimpress Sorry, I'm at a conference at the moment, I'll try and get to it over the next few days.

Reply:
Ok. Finally had a chance to look at this properly. I was confusing this sniff with the sister-sniff `DisallowAlternativePHPTags`. I'm sending you a PR with a very minor addition to this PR.

Other than that, I noticed another issue regarding the `DisallowShortOpenTag` sniff (and it's sister), but that should be handled in a separate PR - see #1784.

And another issue regarding the testing of the `DisallowAlternativePHPTags` sniff, but, again, that should be handled in another PR - see #1786.